### DISCUSSION TITLE
How to install react

### DISCUSSION THREAD

**Discussion by Mad-Man-Dog** [Mad-Man-Dog/JSUtilityKit#4] [Q&A] [answered]
What are the steps involved in installing and setting up React.js for web development? ReactJs Book

**Comment by ayeshanweerasuriya** [comment, marked as answer]
npm create-react-app {application name}

**Reply by Mad-Man-Dog** [reply]
fantastic

**Reply by ayeshanweerasuriya** [reply]
thank you

**Comment by iamsirjan** [comment]
npm create vite@latest

enter this  on command 
before that you need to have node js installed on your computer and also install npm globally 

using vite is better than create-react-app and also fast .
